### PR TITLE
Skip explicitAttrs No Lib

### PR DESCRIPTION
**Pull request by mapedraza**
Modify tests to skip this cases in JSON agent untill the following issue is fixed:
https://github.com/telefonicaid/iotagent-json/issues/782

### CODE REVIEW

File: test/functional/testCases.js [hunk 1]
```diff
@@ -2424,6 +2424,7 @@ const testCases = [
             {
                 shouldName: 'C - WHEN sending only b IT should store only [{object_id:b}] attrs into Context Broker',
                 type: 'single',
+                skip: '!lib', // FIXME: https://github.com/telefonicaid/iotagent-json/issues/782
```
Comment (fgalan) [inline test/functional/testCases.js:2427]: The usage of `lib` is documented in https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/functional/README.md, but not the usage of `!lib`. Maybe that document should be improved to explain the `!lib` case.

Comment (mapedraza) [reply test/functional/testCases.js:2427]: You are right. Anyway, I think the best is to create a separated PR.

Comment (fgalan) [reply test/functional/testCases.js:2427]: Let's do it and merge both at the same time?

Comment (mapedraza) [reply test/functional/testCases.js:2427]: https://github.com/telefonicaid/iotagent-node-lib/pull/1532

**Pull request merged**